src/Appwrite/Platform/Tasks/Specs.php

```diff
@@ -49,11 +49,10 @@ public function __construct()
             ->desc('Generate Appwrite API specifications')
             ->param('version', 'latest', new Text(16), 'Spec version', true)
             ->param('mode', 'normal', new WhiteList(['normal', 'mocks']), 'Spec Mode', true)
-            ->inject('register')
-            ->callback(fn (string $version, string $mode, Registry $register) => $this->action($version, $mode, $register));
+            ->callback([$this, 'action']);
```

Comment:
lets use the alternate syntax - $this->action(...)

Reply:
Updated